tests/tools/kopiarunner/kopia_snapshotter.go

```diff
@@ -395,7 +395,7 @@ func (ks *KopiaSnapshotter) ConnectOrCreateRepoWithServer(serverAddr string, arg
 	var cmdErr error
 
 	if cmd, cmdErr = ks.CreateServer(serverAddr, serverArgs...); cmdErr != nil {
-		return nil, "", cmdErr
+		return nil, "", errors.Wrap(cmdErr, "CreateServer failed")
 	}
 
 	if err := certKeyExist(context.TODO(), tlsCertFile, tlsKeyFile); err != nil {
```

Comment:
I suggest we make our lives easier in the future, perhaps. How about we also leave a breadcrumb that maybe the repo open is taking too long?
```go
	if err := certKeyExist(context.TODO(), tlsCertFile, tlsKeyFile); err != nil {
		if buf, ok := cmd.Stderr.(*bytes.Buffer); ok 
			// If the STDERR buffer does not contain any obvious error output,
			// it is possible the async server creation above is taking a long time
			// to open the repository, and we timed out waiting for it to write the TLS certs.
			// Please see <add a link to some context here?> for more info.
			log.Print("failure in certificate generation:", buf.String())
		}
```

Reply:
Good point. Done.